### DISCUSSION TITLE
Home in center of plate?

### DISCUSSION THREAD

**Discussion by bakaneko59** [Jyers/Marlin#216] [Q&A] [answered]
I just installed a precompiled bin (E3V2-UBL-NoProbe-5x5-v4.2.2.bin) and my machine now homes in the center of the plate.  Is this intentional or did I screw something up?  Apart from that the build seems to work just fine.

**Comment by Jyers** [comment, marked as answer]
That is correct, but it doesn't need to be that way. Must have been an accidental carryover from the BL Touch version. I'll probably disable it in the next release

**Reply by cobyn** [reply]
nice thank you, the discharge from the preheat is getting stuck during this step. had to clean it up before it starts. removing that will great help.
